script/PostChecker.sol

```diff
@@ -215,16 +217,37 @@ contract PostChecker is
       assertEq(stakingVesting.fastFinalityRewardPercentage(), newPercentage, "REP10 percentage not match");
 
       _randomlySubmitBlockReward({ validatorSet: validatorSet, txFee: 0.3 ether });
+      _randomlySubmitL2BlockReward({ validatorSet: validatorSet, txFee: 0.4 ether });
     }
 
     console.log(StdStyle.green("Cheat fast forward to 1 epoch ...\n"));
     LibWrapUpEpoch.wrapUpEpoch();
   }
 
+  function _randomlySubmitL2BlockReward(
```

Comment:
missing check the state change accurately

Reply:
ok